samples/wifi/radio_test/sample_description.rst

```diff
@@ -504,6 +510,7 @@ Testing
               The captured samples will vary from run to run.
             * The capture is taken after WLAN packet detection, so it will not have the first few samples in the first WLAN packet.
             * Smaller packets should be used so that multiple packets can be seen in the capture.
+            * In clean environment packet detection will fail.
```

Comment:
1. In **RF** clean environment
 2. Instead of saying "will fail" it is better to say "does not happen". ???